SPECS/python-pytest-mock/python-pytest-mock.spec

```diff
@@ -0,0 +1,186 @@
+%global pypi_name pytest-mock
+%global file_name pytest_mock
+
+Summary:        Thin-wrapper around the mock package for easier use with py.test
+Name:           python-%{pypi_name}
+Version:        3.5.1
+Release:        3%{?dist}
+License:        MIT
+URL:            https://github.com/pytest-dev/pytest-mock/
+Vendor:         Microsoft Corporation
+Distribution:   Mariner
+Source0:        https://files.pythonhosted.org/packages/24/1b/ddad49c762bddfe3cb61c8ba61349701afd584b84ff4189721bcba624598/%{pypi_name}-%{version}.tar.gz
```

Comment:
>https://files.pythonhosted.org/packages/24/1b/ddad49c762bddfe3cb61c8ba61349701afd584b84ff4189721bcba624598/%{pypi_name}-%{version}.tar.gz [](http://example.com/codeflow?start=16&length=137)

Could you use the [tarball from the project's repo](https://github.com/pytest-dev/pytest-mock/archive/refs/tags/v3.5.1.tar.gz)?

Reply:
Same request for other such cases.

Reply:
For many of these the tar from the GitHub does not match the source from pypi. I was using pypi to keep with parity and because that is what pip pulls from typically.

Reply:
What do you mean by "does not match"? Are the files inside the tarballs completely different? If they are not or the package still builds correctly, then I'd still prefer the GitHub sources, because they are easier to update for newer versions, since they don't contain the hash component.